bridgesample/src/main/java/com/livefront/bridgesample/scenario/activity/LargeDataActivity.kt

```diff
@@ -50,13 +50,13 @@ class LargeDataActivity : BridgeBaseActivity() {
 
         fun getArguments(
             activity: LargeDataActivity
-        ): LargeDataActivityArguments = activity
+        ): LargeDataActivityArguments? = activity
                 .intent
                 .getParcelableExtra(ARGUMENTS_KEY)
 
         fun getNavigationIntent(
             context: Context,
-            arguments: LargeDataActivityArguments
+            arguments: LargeDataActivityArguments?
```

Comment:
Why are you making this nullable exactly? Whatever the reason, we should find a way around this since we want to strictly enforce (somehow) that this is non-null.